packages/lib/src/vis/tiles/TiledHeatmap.tsx

```diff
@@ -23,14 +23,20 @@ function TiledHeatmap(props: Props) {
   } = props;
 
   const { visSize } = useAxisSystemContext();
-  const { xDataPerPixel, yDataPerPixel } = useDataPerPixel();
+  const { xNormPerPixel, yNormPerPixel } = useNormalizedResolution();
 
-  const { baseLayerIndex } = api;
+  const { baseLayerIndex, baseLayerSize } = api;
+  const { width, height } = baseLayerSize;
+
+  const itemPerPixel = Math.max(
```

Comment:
This is now the resolution given as the number of image array items per screen pixel